composer.json

```diff
@@ -68,7 +68,7 @@
         "symfony/symfony": "2.8.10 || 3.4.12",
         "symfony/phpunit-bridge": "2.8.10",
         "symfony/monolog-bundle": "2.8.10",
-        "jackalope/jackalope": "1.2.8 || 1.3.0 || 1.3.1",
+        "jackalope/jackalope": "1.2.8 || 1.3.0 || 1.3.1 || 1.3.6",
```

Comment:
Do they know and do something about this issue?

Reply:
Didn't got any feedback yet from @dbu or @middlebrain

Reply:
created also an issue in jackrabbit repository as there the build fails with the same error: https://github.com/jackalope/jackalope-jackrabbit/issues/156